code/game/objects/items/mop.dm

```diff
@@ -52,6 +52,8 @@
 		return CLEAN_BLOCKED
 	if(reagents.has_reagent(amount = 1, chemical_flags = REAGENT_CLEANS))
 		return CLEAN_ALLOWED
+	else
```

Comment:
There's no need for the else statement.

remove it and In the return statement below change CLEAN_BLOCKED to CLEAN_ALLOWED